get_latest/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<head>
+<title>Get QField</title>
+</head>
+<body>
+<script type="text/javascript">
+const API_LATEST_RELEASE_URL = 'https://api.github.com/repos/opengisch/QField/releases/latest';
+const BROWSER_LATEST_RELEASE_URL = 'https://github.com/opengisch/QField/releases/latest';
+try {
+  const queryParams = new Proxy(new URLSearchParams(window.location.search), {
+    get: (searchParams, prop) => searchParams.get(prop),
+  });
+  if (!queryParams.platform) {
+    throw new Error('No "platform" parameter provided!');
+  }
+  fetch(API_LATEST_RELEASE_URL)
+    .then((response) => {
+      if (!response.ok) throw new Error();
+      return response.json();
+    })
+    .then(json => {
+      var downloadUrl = "https://github.com/opengisch/QField/releases/download/" + json['tag_name'] + "/qfield-" + json['tag_name'] + "-";
+      if (queryParams['platform'] == "windows") {
+        downloadUrl += "windows-x64.exe";
+      } else if (queryParams['platform'] == "linux") {
+        downloadUrl += "linux-x64.AppImage";
+      }
```

Comment:
Note with this implementation difference it means you have to edit this file any time you want to download a new platform binary. Why don't you pass the whole suffix of the asset name, as suggested in my original comment?

Reply:
@suricactus , I thought qfield.org/get_latest/?platform={windows,linux} were nicer links than qfield.org/get_latest/?platform={windows-x64.exe,linux-x64.AppImage}.

I'd rather have to do one edit whenever we add a new platform (which would be once per few years? :) ) than having longer and slightly more "intimidating" URLs.

Reply:
we experience different laziness levels  :)

Reply:
A bit too late, but just to satisfy my laziness and your less intimidating URLs, we should have added `else { downloadUrl = queryParams['platform'] ;  }`